Improve info pane

mwoehlke-kitware opened the pull request:
Avoid superfluous DOM elements. Improve "hint" message text and appearance. Use text-relative dimensions for things that need to be relative to the text size. Use flex to show hint at bottom rather than absolute positioning. Fix overflow in info pane (#21).

mwoehlke-kitware committed:
Improve info pane

Avoid superfluous DOM elements. Improve "hint" message text and
appearance.

mwoehlke-kitware committed:
Fix overflow in info pane

Add necessary CSS to scroll info pane if the contents don't fit, rather
than causing the panel to grow beyond the screen height.

Fixes #21.

aashish24 commented on web_external/InfoView/style.styl:
```diff
@@ -7,26 +7,29 @@
+
     &.geometry
-      padding-left 20px
+      padding-left 1em
```
@mwoehlke-kitware lets not change px to em for now until we have a follow up discussions.

mwoehlke-kitware committed:
Use flex layout for info pane

Change layout rules to use flex to position the hint at the bottom of
the info pane rather than absolute position. This makes it possible to
size the main content to the remaining space, which will also make it
possible to add proper scrolling behavior.

aashish24 reviewed (approved):
LGTM if @matthewma7 okays this.

matthewma7 reviewed (approved):
Look good to me too
